log_test.go

```diff
@@ -111,7 +111,7 @@ func TestLogFormat(t *testing.T) {
 	buf.Reset()
 
 	// Fatal log
-	var hadExit = false
+	hadExit := false
```

Comment:
```suggestion
	var hadExit bool
```

Reply:
Better?

Reply:
Yes, this seems bit better than the earlier one. :)